app/upgrades/v16/upgrades.go

```diff
@@ -40,6 +41,9 @@ func CreateUpgradeHandler(
 			return nil, err
 		}
 
+		// set params for CL keeper before we call internal logic
+		keepers.ConcentratedLiquidityKeeper.SetParams(ctx, cltypes.DefaultParams())
+
```

Comment:
We shouldn't need this as the parameters must be set by the `RunMigrations` function above that runs `InitGenesis()`

I think the solution to the problem is only the second part of this PR - setting allowed swap fee

```suggestion
```